teams/3/list-examples.html

```diff
@@ -0,0 +1,152 @@
+                        training smarter, not harder.
+                    </p>
+                </li>
+                <li class="text-xl-med">
+                    <img src="./assets//charge.svg" alt="charge" />
+                    <h3>Measure recovery</h3>
+                    <p>
+                        The most advanced sleep tracking technology available
+                        today. Measure and track your recovery to unlock your
+                        greatest potential.
+                    </p>
+                </li>
+                <li class="text-xl-med">
+                    <img src="./assets//phone.svg" alt="phone" />
+                    <h3>Tech that evolves with you</h3>
+                    <p>
+                        Know where your strengths lie and where you can improve.
+                        Untitled provides the latest tech with a steady stream
+                        of new features.
+                    </p>
+                </li>
+                <li class="text-xl-med">
+                    <img src="./assets//social.svg" alt="social" />
+                    <h3>Unrivalled community</h3>
+                    <p>
+                        Join teams in the app with friends, family, and
+                        like-minded fitness enthusiasts. Create custom teams
+                        based on activities and interests.
+                    </p>
+                </li>
+            </ul>
+        </section>
+        <section>
+            <h2>Example number 2</h2>
+            <ul class="ut-list-cols-3">
+                <li class="text-md-reg">
+                    <img src="./assets//notion.svg" alt="notion icon" />
+                    <h3>Notion integration</h3>
+                    <p>
+                        Work faster and smarter by integrating directly with
+                        Notion, right in the app.
+                    </p>
+                    <a>View integration</a>
+                </li>
+                <li class="text-md-reg">
+                    <img src="./assets/slack.svg" alt="slack icon" />
+                    <h3>Slack integration</h3>
+                    <p>
+                        Work faster and smarter by integrating directly with
+                        Slack, right in the app.
+                    </p>
+                    <a>View integration</a>
+                </li>
+                <li class="text-md-reg">
+                    <img
+                        src="./assets/google-drive.svg"
+                        alt="google drive icon"
+                    />
+                    <h3>Google Drive integration</h3>
+                    <p>
+                        Work faster and smarter by integrating directly with
+                        Google Drive, right in the app.
+                    </p>
+                    <a>View integration</a>
+                </li>
+                <li class="text-md-reg">
```

Comment:
please do not use text-red or padding-something classes. Create a class for the li with a more project-related name. And if it is not a class intended to be used with a parent, should be prefixed,

Reply:
Sorry Grace.  I don't understand. Typography class names are not made by me. Do you want me to re-create those typography classes for each kind of list item in the project?

Reply:
Not at all, typography classes are ready to use and made by team 1. You just need to use those classes in your texts. Please align with team 1 so they show you how to use the classes.

Reply:
I think he is using the typography classes correctly now.